packages/appstore/src/components/elements/text/balance-text.scss

```diff
@@ -5,22 +5,12 @@
 
     &__text {
         &--demo {
-            color: var(--text-profit-success);
+            color: var(--status-info) !important;
```

Comment:
Is using !important necessary here ? We did not need it for the previous color.

Reply:
I think it's not needed, just removed it 👌🏼